src/test/scala/uk/gov/hmrc/api/service/MonthlyReturnsSubmissionService.scala

```diff
@@ -0,0 +1,188 @@
+  ): LISAClosurePayload = LISAClosurePayload(
+    accountNumber,
+    nino,
+    "First24998",
+    "LISA Closure model",
+    "Last24998",
+    "1980-01-22",
+    "LIFETIME_STOCKS_AND_SHARES",
+    false,
+    "2025-06-01",
+    2500.23,
+    10000.12,
+    "2025-01-22",
+    "2025-03-22",
+    "CLOSED",
+    5000.56,
+    3000.56
+  )
+
+  def getLISATransferAndClosurePayload(
+    nino: String,
+    accountNumber: String
+  ): LISATransferAndClosurePayload = LISATransferAndClosurePayload(
+    accountNumber,
+    nino,
+    "First24999",
+    "LISA Transfer & Closure model",
+    "Last24999",
+    "1980-01-22",
+    "LIFETIME_STOCKS_AND_SHARES",
+    true,
+    "2025-06-01",
+    2500.23,
+    10000.12,
+    "1234567",
+    1200.34,
+    "2025-01-22",
+    "2025-03-22",
+    "TRANSFERRED_IN_FULL",
+    5000.56,
+    3000.56
+  )
+
+  def getLISATransferPayload(
+    nino: String,
+    accountNumber: String
+  ): LISATransferPayload = LISATransferPayload(
+    accountNumber,
+    nino,
+    "First25000",
+    "LISA Transfer model",
+    "Last25000",
+    "1980-01-22",
+    "LIFETIME_CASH",
+    true,
+    "2025-06-01",
+    2500.23,
+    10000.12,
+    "1234567",
+    1200.34,
+    "2025-01-22",
+    5000.56,
+    3000.56
+  )
+
+  def getSISANewSubscriptionPayload(
+    nino: String,
+    accountNumber: String
+  ): SISANewSubscriptionPayload = SISANewSubscriptionPayload(
+    accountNumber,
+    nino,
+    "First25001",
+    "SISA new subscription model",
+    "Last25001",
+    "1980-01-22",
+    "STOCKS_AND_SHARES",
+    false,
+    "2025-06-01",
+    2500.23,
+    10000.12,
+    false
+  )
+
+  def getSISATransferPayload(
+    nino: String,
+    accountNumber: String
+  ): SISATransferPayload = SISATransferPayload(
+    accountNumber,
+    nino,
+    "First25002",
+    "SISA Transfer model",
+    "Last25002",
+    "1980-01-22",
+    "CASH",
+    true,
+    "2025-06-01",
+    2500.23,
+    10000.12,
+    "1234567",
+    1200.34,
+    true
+  )
+
+  def postMonthlyReturnsSubmission(
+    isaManagerReference: String,
+    returnId: String,
+    headers: Map[String, String]
+  ): StandaloneWSResponse = {
+    val lisaNewSubscriptionPayload    =
+      getLISANewSubscriptionPayload(RandomDataGenerator.nino(), RandomDataGenerator.generateSTDCode())
+    val lisaClosurePayload            = getLISAClosurePayload(RandomDataGenerator.nino(), RandomDataGenerator.generateSTDCode())
+    val lisaTransferAndClosurePayload =
+      getLISATransferAndClosurePayload(RandomDataGenerator.nino(), RandomDataGenerator.generateSTDCode())
+    val lisaTransferPayload           = getLISATransferPayload(RandomDataGenerator.nino(), RandomDataGenerator.generateSTDCode())
+    val sisaNewSubscriptionPayload    =
+      getSISANewSubscriptionPayload(RandomDataGenerator.nino(), RandomDataGenerator.generateSTDCode())
+    val sisaTransferPayload           = getSISATransferPayload(RandomDataGenerator.nino(), RandomDataGenerator.generateSTDCode())
```

Comment:
Up to you: I'm wondering if the setup of the models could be done by having these values as default parameters on the case classes themselves, including generation of NINO and STD. Particularly if the same values are going to be used frequently in other tests (I don't know if this is the case).